docs/data/system/components/box/box.md

```diff
@@ -9,65 +9,47 @@ githubLabel: 'component: Box'
 
 <p class="description">The Box component serves as a wrapper component for most of the CSS utility needs.</p>
 
-The Box component packages [all the style functions](/system/getting-started/the-sx-prop/) that are exposed in `@mui/system`.
-
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
-## Example
-
-[The palette](/system/palette/) style function.
+## Introduction
 
-## The `sx` prop
+The Box component is a generic container for grouping other components.
+It's a fundamental building block when building with MUI component libraries—you can think of it as a `<div>` with special features (like access to your app's theme and the [`sx` prop](/system/getting-started/the-sx-prop/)).
 
-All system properties are available via the [`sx` prop](/system/getting-started/the-sx-prop/).
-In addition, the `sx` prop allows you to specify any other CSS rules you may need. Here's an example of how you can use it:
+## Basics
 
-{{"demo": "BoxSx.js", "defaultCodeOpen": true }}
-
-## Overriding MUI components
```

Comment:
@samuelsycamore — ended up removing this section altogether as I felt like the brief `component` prop on the Basics section already introduced this whole concept well. I'm not fully sure what this previous section targeted with stuff like "...sometimes you have to target the underlying DOM element", the code changes exposed here didn't necessarily communicate that to me... let me know what you think about it!

Reply:
Good call!